src/unity/lib/unity_sframe.cpp

```diff
@@ -1673,7 +1673,101 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
       ew << ss.str();
     }
 
-    auto getRows = [self, &reader, &ew, &column_names, &empty_tz, &image_queue](size_t start, size_t end) {
+    std::function<std::string(flexible_type&)> listHelper;
+    listHelper = [&listHelper, &empty_tz](flexible_type& value) {
```

Comment:
Can we come up with a better name than `listHelper`? How about `escapeForTable`? (Is this only for the accordion, only for inline display in the table, or both? Let's pick a name that communicates the answer to that question...)

Reply:
Just for the inline display in the table. How's `escapeForInlineTable`?

Reply:
I find the word `inline` confusing. I think `escapeForTable` is fine.